### DISCUSSION TITLE
minDisplayWidth / maxDisplayWidth

### DISCUSSION THREAD

**Discussion by adgad** [Financial-Times/content-tree#22] [Team Posts]
These were initially provided by Methode to distinguish the desktop / mobile image. In hindsight, it doesn't really make sense for the CMS to publish these widths, and I don't believe the values themselves are actually used directly.

We do now have the `format` field which also provides `desktop` / `mobile` (barring any changes resulting from other issues). Are `minDisplayWidth` / `maxDisplayWidth` redundant?

**Comment by atanasdinov** [comment]
Both `minDisplayWidth` and `maxDisplayWith` are not validated at all and pass validation as additional fields of the `members` field in the image set schema.

If they're not required anymore, I'd recommend against going through the old content and removing them. Ignoring these seems like our best option here.
